packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -72,6 +72,10 @@ spec:
         image: {{ template "system_default_registry" . }}{{ .Values.global.proxy.repository }}:{{ .Values.global.proxy.tag }}
       proxy_init:
         image: {{ template "system_default_registry" . }}{{ .Values.global.proxy_init.repository}}:{{ .Values.global.proxy_init.tag }}
+      {{- if .Values.global.defaultPodDisruptionBudget.enabled }}
+      defaultPodDisruptionBudget:
+        enabled: {{ .Values.global.defaultPodDisruptionBudget.enabled }}
```

Comment:
I thought this was a min/max setting, what exactly does this do ?

Reply:
```
  # enable pod disruption budget for the control plane, which is used to
  # ensure Istio control plane components are gradually upgraded or recovered.
  defaultPodDisruptionBudget:
    enabled: true
    # The values aren't mutable due to a current PodDisruptionBudget limitation
    # minAvailable: 1
```

which enabled this https://github.com/istio/istio/blob/master/manifests/charts/gateways/istio-ingress/templates/poddisruptionbudget.yaml